tests/unittests/gradCheckTest.cpp

```diff
@@ -199,11 +200,29 @@ static void gradientCheckGroupConv(size_t numInputChan, size_t group) {
   performGradCheck(IP, result, A, Ex, &inputs, &outputs, 0.001, 0.004);
 }
 
-TEST(Network, gradientCheckConv) { gradientCheckGroupConv(1, 1); }
+TEST(Network, gradientCheckConv) {
+  gradientCheckGroupConv(1, 1, BackendKind::Interpreter);
+}
+
+TEST(Network, gradientCheckDepthwiseConv) {
+  gradientCheckGroupConv(4, 4, BackendKind::Interpreter);
+}
+
+TEST(Network, gradientCheckGroupConv) {
+  gradientCheckGroupConv(4, 2, BackendKind::Interpreter);
+}
 
-TEST(Network, gradientCheckDepthwiseConv) { gradientCheckGroupConv(4, 4); }
+TEST(Network, gradientCheckConvCPU) {
```

Comment:
Consider using a parameterized test fixture (`TEST_P`) for the different backends, like we use in OperatorTest and MLTest.  It can be a separate PR if you'd like.

Reply:
that was fast :)